command/ca/init.go

```diff
@@ -523,9 +523,9 @@ func initAction(ctx *cli.Context) (err error) {
 	if pkiOnly {
 		pkiOpts = append(pkiOpts, pki.WithPKIOnly())
 	} else {
-		ui.Println("What DNS names or IP addresses would you like to add to your new CA?",
+		ui.Println("What DNS names and IP addresses will clients use to reach your CA?",
```

Comment:
This makes me think I need DNS names **and** IP addresses.

Reply:
good call, changed it back to `or`